package-lock.json

```diff
@@ -1505,25 +1505,29 @@
           "dependencies": {
             "abbrev": {
               "version": "1.1.1",
-              "bundled": true,
+              "resolved": false,
+              "integrity": "sha512-nne9/IiQ/hzIhY6pdDnbBtz7DjPTKrY00P/zvPSm5pOFkl6xuGrGnXn/VtTNNfNtAfZ9/1RtehkszU9qcTii0Q==",
```

Comment:
I don't think the rest of these changes to package-lock should be part of this PR, just the version number above (though we probably do need to update dependencies separately... it's been a while).

Reply:
Copy that. Just reverted it. 👍